### ISSUE TITLE
Invalid podman arguments in makefile

### ISSUE THREAD

**Issue by mshriver** [app-sre/qontract-reconcile#3652]
The Makefile includes automatic use of podman, but includes an option that is not accepted by podman.

https://github.com/app-sre/qontract-reconcile/blob/master/Makefile#L68

This argument is not valid for podman:
```
--add-host=host.docker.internal:host-gateway
```

And creates the following failure mode when `dev-reconcile-loop` is used:
```
Error: invalid IP address in add-host: "host-gateway"
```

Currently playing with what would be valid arguments for podman runtime. It ties into qontract-server makefile as well, since that runs the server with direct port bindings. Maybe both using an expected podman network would be a preferable solution, although it looks like the argument isn't needed at all when running in podman, or the `--network host` argument can be used.

**Comment by maorfr** [comment]
related to https://issues.redhat.com/browse/APPSRE-7028

**Comment by maorfr** [comment]
fwiw, you can `pip install qontract-reconcile`

**Comment by mshriver** [comment]
@maorfr yeah but I'm trying to break it and be a PITA and use podman :laughing:
